src/components/ObjectViewer/ObjectViewer.vue

```diff
@@ -20,10 +21,24 @@
 
 <script setup lang="ts">
 import config from "@/config";
+import { onMounted } from "vue";
 
 defineProps<{
   fileHandlerId: string | null;
 }>();
+
+const emit = defineEmits<{
+  (eventName: "objectViewLoad");
+}>();
+
+onMounted(() => {
+  console.log("ObjectViewer mounted");
```

Comment:
Do we want to leave these console logs? I'm fine with having some standard debug logs in the codebase if we think it's useful, but we should do some standardize prefixing if they're gonna be in prod

Reply:
Nope. We can remove.